src/client/components/PlainMarkdownInput/plugins/slate-autocomplete-plugin/AutocompleteWidget.js

```diff
@@ -79,7 +81,7 @@ class AutocompleteWidget extends React.Component {
     }
   };
 
-  handleSelectItem = (index) => {
+  handleSelectItem = index => _ => {
```

Comment:
Not sure that `handleSelectItem = index => _ =>` is more clear than `handleSelectItem = (index) =>`.